src/components/form/DateField.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import LuxonUtils from '@date-io/luxon';
+import { LocalizationProvider } from '@material-ui/pickers';
+import { action } from '@storybook/addon-actions';
+import { boolean, select, text } from '@storybook/addon-knobs';
+import React from 'react';
+import { Form } from 'react-final-form';
+import { DateField as DF } from './DateField';
+
+export default { title: 'Components/DateField' };
+
+export const DateField = () => (
+  <LocalizationProvider dateAdapter={LuxonUtils}>
+    <Form onSubmit={action('onSubmit')}>
+      {({ handleSubmit }) => (
+        <form onSubmit={handleSubmit}>
+          <DF
+            name="DateField"
+            disabled={boolean('Disabled', false)}
+            fullWidth={boolean('Full Width', false)}
+            variant={select(
+              'Variant',
+              ['standard', 'outlined', 'filled'],
+              'standard'
+            )}
```

Comment:
Let's remove `variant`s in all the stories. We default to `filled` variant in our theme and probably don't want that to be changed on a per use basis.